components/dealInDetails/sliderAndTimerComponents/LimitedOffer.tsx

```diff
@@ -0,0 +1,221 @@
+      <Box
+        gap={0.5}
+        sx={{
+          ...all_center,
+          height: "auto",
+          width: "40%",
+          display: "flex",
+          flexDirection: "column",
+        }}
+      >
+        <Typography sx={{ fontSize: theme.typography.subtitle1.xl }}>
+          Clicks
+        </Typography>
+
+        <Typography sx={{ fontSize: theme.typography.subtitle1.xl }}>
+          {dealAndStoreAllDetails?.clicks}
+        </Typography>
+      </Box>
+
+      <Divider
+        sx={{
+          bgcolor: theme.palette.grey[100],
+          width: "100%",
+        }}
+      >
+        {" "}
+      </Divider>
+
+      <Box
+        component={"div"}
+        sx={{
+          ...all_center,
+          height: "auto",
+          width: "100%",
+          display: "flex",
+          flexDirection: "column",
+          mt: { xl: "1rem" },
+        }}
+      >
+        <Box
+          component={"div"}
+          gap={3}
+          sx={{
+            ...all_center,
+            height: "auto",
+            width: "80%",
+            display: "flex",
+            flexDirection: "column",
+          }}
+        >
+          <Typography sx={{ fontSize: theme.typography.subtitle1.xl }}>
+            Time Left - Limited Offer!
+          </Typography>
+
+          <Box
+            component={"div"}
+            sx={{
+              height: "auto",
+              width: "100%",
+              display: "flex",
+              justifyContent: "space-between",
+            }}
+          >
+            <Box component={"div"} sx={{ ...counterBoxStyle }}>
+              <Typography>{days}</Typography>
+              <Typography>Days</Typography>
+            </Box>
+
+            <Box component={"div"} sx={{ ...counterBoxStyle }}>
+              <Typography>{hours}</Typography>
+              <Typography>Hours</Typography>
+            </Box>
+
+            <Box component={"div"} sx={{ ...counterBoxStyle }}>
+              <Typography>{minutes}</Typography>
+              <Typography>Minutes</Typography>
+            </Box>
+
+            <Box component={"div"} sx={{ ...counterBoxStyle }}>
+              <Typography>{seconds}</Typography>
+              <Typography>Seconds</Typography>
+            </Box>
+          </Box>
+        </Box>
+        <Divider
+          sx={{
+            bgcolor: theme.palette.grey[100],
+            width: "100%",
+            mt: { xl: "1rem" },
+          }}
+        >
+          {" "}
+        </Divider>
+      </Box>
+
+      <Box
+        component={"div"}
+        sx={{ ...all_center, height: "auto", width: "100%" }}
+      >
+        <Box
+          component={"div"}
+          sx={{
+            height: "auto",
+            width: "80%",
+            display: "flex",
+            justifyContent: "space-evenly",
+            pb: "2rem",
+          }}
+        >
+          <Box
+            component={"div"}
+            sx={{ ...socialMediaBoxStyyle, bgcolor: "#4267B2" }}
+          >
+            {" "}
```

Comment:
Why you have added `{""}` ?

Reply:
Sir prettier add it by default during formatting.